### PR TITLE
add code to fix rakudo repo issue #1282

### PR DESCRIPTION
**Pull request by tbrowder**
allows escaping visual column separators

new tests are added in roast PR #361

### CODE REVIEW

File: src/Perl6/Pod.nqp [hunk 1]
```diff
@@ -327,6 +327,39 @@ class Perl6::Pod {
...
+	my $ret := ''; # the cleaned string (i.e., without comments)
+	my $idx := nqp::index($s, '<'); # find the possible beginning of a formatting code
+        if $idx < 0 {
+            # no '<' found so return the simple string
+            return $s;
+        }
+
+	while $idx > -1 {
+            my $idx2 := nqp::index($s, '>', $idx+2); # and the end
+            nqp::die("FATAL:  Non-existent closing '>' for inline pod comment in string '$s' in Table $table_num") if $idx2 < 0;
```
Comment (samcv) [inline src/Perl6/Pod.nqp]: Maybe: "Couldn't find terminator '>' for inline pod comment..." so it matches closer to the wording we use for other error messages.

File: src/Perl6/Pod.nqp [hunk 2]
```diff
@@ -327,6 +327,39 @@ class Perl6::Pod {
...
+            $ret := nqp::concat($ret, $s0);
+
+            # throw away the orig string up to the end of the found comment
+            $s := nqp::substr($s, $idx2+1); # the trailing chunk
+            # look for another comment in the remaining string
+            $idx := nqp::index($s, 'Z<');
+	}
+
+	# make sure we use up a non-empty string end
+	if $s {
```
Comment (samcv) [inline src/Perl6/Pod.nqp]: Could maybe be this to be more compact:
```perl6
 $ret := nqp::concat($ret, $s) if $s;
```

File: src/Perl6/Pod.nqp (hunk 1 above)
Comment (tbrowder) [reply src/Perl6/Pod.nqp]: ok, will change...

File: src/Perl6/Pod.nqp (hunk 2 above)
Comment (tbrowder) [reply src/Perl6/Pod.nqp]: yep...

**Comment by tbrowder** [comment]
Thanks for the review, approval, and merge.

The twine data structure is a weird array interleaving text and Pod::FormattingCode objects.  It's not well documented.

**Pull request merged**